com.ibm.wala.cast.python.jython3/source/com/ibm/wala/cast/python/loader/Python3Loader.java

```diff
@@ -94,15 +101,25 @@ public ConstantFoldingRewriter createCAstRewriter(CAst ast) {
             return new ConstantFoldingRewriter(ast) {
               @Override
               protected Object eval(CAstOperator op, Object lhs, Object rhs) {
+                String s = lhs + " " + op.getValue() + " " + rhs;
```

Comment:
You could declare `s` inside the lambda body so it's only computed if logging is enabled